### ISSUE TITLE
Underscore v1.13.5 install error

### ISSUE THREAD

**Issue by ODGodinho** [jashkenas/underscore#2970]
I'm getting the following error with the new version v1.13.5

> yarn install

Exit code: 127
Command: patch-package
Arguments: 
Directory: /home/dev123/Documentos/projetos/MyProject/node_modules/underscore
Output:
/bin/sh: 1: patch-package: not found
    at ProcessTermError.ExtendableBuiltin (/home/dev123/.nvm/versions/node/v16.17.0/lib/node_modules/yarn/lib/cli.js:721:66)
    at ProcessTermError.MessageError (/home/dev123/.nvm/versions/node/v16.17.0/lib/node_modules/yarn/lib/cli.js:750:123)
    at new ProcessTermError (/home/dev123/.nvm/versions/node/v16.17.0/lib/node_modules/yarn/lib/cli.js:790:113)
    at ChildProcess.<anonymous> (/home/dev123/.nvm/versions/node/v16.17.0/lib/node_modules/yarn/lib/cli.js:25787:17)
    at ChildProcess.emit (node:events:513:28)
    at maybeClose (node:internal/child_process:1093:16)
    at Socket.<anonymous> (node:internal/child_process:451:11)
    at Socket.emit (node:events:513:28)
    at Pipe.<anonymous> (node:net:757:14)
error /home/dev123/Documentos/projetos/MyProject/node_modules/underscore: Command failed.
Exit code: 127
Command: patch-package
Arguments:

**Comment by iglosiggio** [comment]
You can add patch-package as a devDependency in your project (as a hacky workaround, i don't really like the idea of a dependency patching other dependencies).
This was triggered by https://github.com/jashkenas/underscore/commit/08cb1404eb27b80bcadb87a32fcd515f23f7c9a6

**Comment by mykola-mokhnach** [comment]
I can confirm, we started observing the same error after the recent release:
https://github.com/appium/appium-uiautomator2-driver/actions/runs/3115559702/jobs/5052578876

**Comment by boneskull** [comment]
I think that should be `prepare`, not `postinstall`.

**Comment by boneskull** [comment]
cc @jgonggrijp

**Comment by ODGodinho** [comment]
> You can add patch-package as a devDependency in your project (as a hacky workaround, i don't really like the idea of a dependency patching other dependencies). This was triggered by [08cb140](https://github.com/jashkenas/underscore/commit/08cb1404eb27b80bcadb87a32fcd515f23f7c9a6)

The command did not solve
`yarn add patch-package`
nor putting it directly in package.json and running `yarn`

**Comment by joeycozza** [comment]
Duplicate of https://github.com/jashkenas/underscore/issues/2967

**Comment by iglosiggio** [comment]
> > You can add patch-package as a devDependency in your project (as a hacky workaround, i don't really like the idea of a dependency patching other dependencies). This was triggered by [08cb140](https://github.com/jashkenas/underscore/commit/08cb1404eb27b80bcadb87a32fcd515f23f7c9a6)
> 
> The command did not solve `yarn add patch-package` nor putting it directly in package.json and running `yarn`

Sorry, that advice was mistaken. Installing patch-package globally works. I would advice against doing it... but works.

**Comment by ODGodinho** [comment]
problem solved by putting `resolutions` in my package.json

```json
 {
  "name": ".....",
  "resolutions": {
    "underscore": "1.13.4"
  }
}
```

**Comment by jgonggrijp** [comment]
I just released 1.13.6, hopefully that fixes it. Please close if that is the case.
